test/black-box/sandbox/recipes/root.yaml

```diff
@@ -9,7 +9,8 @@ checkoutScript: |
     echo ok
 buildScript: |
     echo ok
-packageVars: [FOO]
+packageVars: [FOO, EXPECT_UID]
```

Comment:
Is it more reasonable to use packageVarsWeak: [EXPECT_UID] instead of packageVars: [EXPECT_UID], because the UID is just for permission validation and does not affect the result?

Reply:
In principle you're right but in this particular test it is part of the result. If the variable would be weak, Bob would not rebuild the package if just the expected uid changes.